File: src/android/PluginMethodNotValidException.java
package com.bandyer.cordova.plugin;

public class PluginMethodNotValidException extends Exception {

    private static final long serialVersionUID = 123L;

    public PluginMethodNotValidException(String s) {
        super(s);
    }

    public PluginMethodNotValidException(String s, Throwable t) {
        super(s, t);
    }

}

File: src/android/cordova_plugin/src/main/java/com/bandyer/cordova/plugin/PluginMethodNotValidException.kt
package com.bandyer.cordova.plugin

class PluginMethodNotValidException : Exception {

    constructor(s: String) : super(s) {}

    constructor(s: String, t: Throwable) : super(s, t) {}

    companion object {

        private val serialVersionUID = 123L
    }

}
